architecture/adr-001-multi-denomination-rewards.md

```diff
@@ -0,0 +1,88 @@
+# ADR 001: Multi Denomination Rewards
+
+## Changelog
+
+- 2023-05-11: Initial Draft (@joe-bowman)
+
+## Status
+
+PROPOSED
+
+## Abstract
+
+With the advent of replicated security, delegators now accrue rewards in multiple
+denominations. These collected rewards should be distributed proportionally to the
+appropriate qAsset holders for the host zone.
+
+## Context
+
+The current implementation of rewards distribution is such that
+rewards collected in the host zone's staking denom are delegated, minus a
+parameterized fee that is sent, along with the remaining rewards (any non-staking-
+denom fees that have been collected) to the FeeCollectorAccount on Quicksilver to
+be distributed to QCK token holders.
+
+With rewards being accrued in multiple denominations, this proposal asserts that
+these rewards should no longer be directed to QCK token holder, but to the holders
+of the qAssets whose underlying capital generated the rewards.
```

Comment:
```suggestion
With rewards being accrued in multiple denominations, this proposal asserts that
these rewards should no longer be directed to _all_ QCK token holders, but to the holders
of the qAssets whose underlying capital generated the rewards.
```

@joe-bowman this is what you mean right?

How heavyweight will it be to track all of this?

Reply:
No different to participation rewards distribution (+qs2)